animations/effects.json

```diff
@@ -324,5 +324,22 @@
 				}
 			]
 		}
+	],
+	"effect:spell-effect-shields-of-the-spirit-security": [
```

Comment:
Why only the Security feat variant and not the base one?

Reply:
Base effect is an Aura (so should function under the future Aura feature)

Reply:
Its both an aura and an effect.

Reply:
Yeah, I'd think the security effect should be visually different from the normal SoS FX (since it does something different) will include the

Reply:
Include what

Reply:
Also for Prem/vs non prem what's the best way to add that should it just be like?
```
	"contents": [
				{
					"predicate": ["jb2a:free"
					],
								"file": "FREEFILEHERE",
				},
				] ```
				?

Reply:
Ah  sorry, will add shield of spirits base FX in another PR